src/bin/polymer-bundler.ts

```diff
@@ -194,7 +186,7 @@ function stringToRegExp(str: string) {
   return new RegExp(str.replace(/[-\/\\*+?.()|[\]{}]/g, '\\$&'));
 }
 
-options.addedImports = options['add-import'] || [];
+options.basePath = options['abspath'];
```

Comment:
Yeah I think the intent was backwards compatibility.  I've renamed it `--base-path` to correspond to the Bundler constructor option of `basePath`.